prog/hubbard2d_model.h

```diff
@@ -63,8 +64,12 @@ class hubbard2d_model: public quantum_model {
         auto pos = SiteIndexF(x,y);
         auto pos_right = SiteIndexF((x+1)%size_x,y); /*if (x == size_x - 1) pos_right = SiteIndexF(0,y); */
         auto pos_up = SiteIndexF(x,(y+1)%size_y);
+        auto pos_dia = SiteIndexF((x+1)%size_x,(y+1)%size_y);
```

Comment:
Hm, what about the other diagonal of a plaquette?
I mean `SiteIndexF((x+1)%size_x,(y-1)%size_y)`.